man/fi_endpoint.3.md

```diff
@@ -911,6 +911,8 @@ The length of the authorization key in bytes.  This field will be 0 if
 authorization keys are not available or used.  This field is ignored
 unless the fabric is opened with API version 1.5 or greater.
 
+If the domain is opened with FI_AV_AUTH_KEY, auth_key_size is ignored.
```

Comment:
I think I want to change to this to if ep_attr::auth_key_size must be zero.

Reply:
agreed on the null/0 requirements